node/server/server.ts

```diff
@@ -73,12 +89,13 @@ const prepareServer = ({
       // no-op
     }
 
-    const bootstrapFlags = ['--port', port.toString()]
+    const extraDenoFlags = [`--import-map=${importMap.toDataURL()}`]
```

Comment:
The main change here is that rather than computing the import map once the server starts, we do it every time there's a file change. This is because the import map may change when the code adds or remove npm modules.

I've added the `clone()` method to `ImportMap` so that we can keep an import map around with the base entries, and then clone and extend it with new entries on file changes.